### PR TITLE
timeline peeking

### PR DESCRIPTION
**Pull request by orviwan**

### CODE REVIEW

**Comment by cat-haines** [comment]
`.lock-waf_darwin_build` should be in the .gitignore

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "targetPlatforms": [
+      "aplite",
+      "basalt",
+      "chalk",
+      "diorite",
+      "emery"
+    ],
+    "capabilities": []
+  },
+  "name": "Segment Six"
```
Comment (cat-haines) [inline package.json]: I believe name is usually near the top of the `pebble` object

**Comment by cat-haines** [comment]
A fair bit of the hours circle is covered during timeline peek events, is that intentional?

**Comment by cat-haines** [comment]
Screenshot should be in a /screenshots folder

File: src/segment_six.c [hunk 2]
```diff
@@ -68,58 +73,79 @@ static void hour_display_update_proc(Layer *layer, GContext* ctx) {
...
+
+// Event fires once, after obstruction appears or disappears
+static void prv_unobstructed_did_change(void *context) {
+  prv_mark_dirty();
+}
+
+static void prv_main_window_load(Window *window) {
   Layer *window_layer = window_get_root_layer(window);
-  GRect bounds = layer_get_bounds(window_layer);
+  GRect bounds = layer_get_unobstructed_bounds(window_layer);
```
Comment (cat-haines) [inline src/segment_six.c]: Should this be `layer_get_bounds` instead (for drawing background, etc)

**Comment by cat-haines** [comment]
I think this example could be cleaned up and simplified using `graphics_draw_arc` (which is the preferred way to draw arcs)

File: src/segment_six.c (hunk 2 above)
Comment (orviwan) [inline src/segment_six.c]: I believe this is correct. The window background color is already set on init(), then these layers are above that.

**Comment by orviwan** [comment]
> A fair bit of the hours circle is covered during timeline peek events, is that intentional?

I think as a simple example of peek, this is fine.

> I think this example could be cleaned up and simplified using graphics_draw_arc (which is the preferred way to draw arcs)

We could, but this is also a good example of `gpath`. We already have other examples of arcs too.

File: package.json [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {
+	"name": "Segment Six",
```
Comment (cat-haines) [inline package.json]: Use spaces, not a tab

File: README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # segment-six
 
-![screenshot](segment-six-screenshot.png)
+![screenshot](screenshots/segment-six-screenshot.png)
```
Comment (cat-haines) [inline README.md:3]: Missing screenshots folder / screenshot

**Comment by cat-haines** [comment]
:+1: other than missing screenshot / screenshot folder

**Pull request merged**